app/src/js/components/ReingestGranules/BatchReingestCompleteContent.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const maxDisplayed = 10;
+
+const BatchReingestCompleteContent = ({
+  results,
+  error
+}) => {
+  const confirmation = `successfully reingested ${results.length > 1 ? 'these' : 'this'} granule${results.length > 1 ? 's' : ''}`;
+  return (
+    <>
+      {(results && results.length > 0) &&
+       <>
+         <p>{confirmation}</p>
+         <ul>
+           {results.map((result, index) => {
+             if (index < maxDisplayed) {
+               return <li key={index}>{result}</li>;
+             } return <></>;
+           })}
```

Comment:
If the results is a lot longer than `maxDisplayed`, do we still want to be iterating over them? Maybe we can do a for loop where the max is the shorter of `results.length` and `maxDisplayed`?

Reply:
e53a3b4a